build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1997,6 +1997,18 @@ message CacheCapabilities {
   // [BatchUpdateBlobs][build.bazel.remote.execution.v2.ContentAddressableStorage.BatchUpdateBlobs]
   // requests.
   repeated Compressor.Value supported_batch_update_compressors = 7;
+
+  // Maximum size a single CAS blob can have in the remote cache system.
+  // Here is how implementations should interpret this value:
```

Comment:
Nit: this sentence doesn't add any information; it's clear that the remainder describes how to interpret the value.